DotNetLightning.sln.DotSettings

```diff
@@ -7,6 +7,7 @@
 	<s:Boolean x:Key="/Default/UserDictionary/Words/=Bitcoind/@EntryIndexedValue">True</s:Boolean>
 	<s:Boolean x:Key="/Default/UserDictionary/Words/=Bitcoin_0027s/@EntryIndexedValue">True</s:Boolean>
 	<s:Boolean x:Key="/Default/UserDictionary/Words/=blockchain/@EntryIndexedValue">True</s:Boolean>
+	<s:Boolean x:Key="/Default/UserDictionary/Words/=bouncycastle/@EntryIndexedValue">True</s:Boolean>
```

Comment:
should we delete this file from the repo and add it to gitignore? I think it's some IDE thing

Reply:
I sometimes work in different machine so this helps me.
It does not hurt and might help someone who want to work with IDE, so I want to keep it here.